liberty-maven-plugin/src/it/scanner-it/pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.openliberty.tools.it</groupId>
+    <artifactId>scanner-it</artifactId>
```

Comment:
Suggestion to name this `binary-scanner-it` or something related to feature generation so we can be clear which goal this test suite is tied to/intended for

Reply:
Ok, I'll do this in the next drop. I was just trying to keep the name short but plenty of tests have long names.